recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,36 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_MACHINE ?= "xenguest_arm64_defconfig"
```

Comment:
It is a bit strange, that we put defconfig into MACHINE variable. What's the background here? I would expect "xenguest_arm64" or something here

Reply:
maybe something is wrong, but background of your proposal is

Reply:
fixed